libs/util/http.client.js

```diff
@@ -211,6 +211,9 @@ function io(url, options) {
         resp.responseText = body;
 
         if (err) {
+            if (err.message === 'XMLHttpRequest timeout') {
+                err = new Error('XMLHttpRequest timeout: failed to load ' + resp.url + ' in ' + options.timeout + 'ms');
+            }
```

Comment:
Thanks for your response. You are right, those fields are available on the error object, but unfortunately not very helpful for my use-case. 
I see a lot of those timeout errors in the new relic js error report of our production application. Unfortunately new relic only shows the error message and stack trace and both information don't help at all to narrow down the cause of the error.
Maybe I could somehow wrap all `context.service` calls from `fluxible-plugin-fetchr` instead, I guess that would also solve my issue.